src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.cs

```diff
@@ -59,7 +59,7 @@ public async Task<Document> AddSourceToAsync(Document document, ISymbol symbol,
         /// <summary>
         /// provide formatting rules to be used when formatting MAS file
         /// </summary>
-        protected abstract IEnumerable<IFormattingRule> GetFormattingRules(Document document);
+        protected abstract IEnumerable<Formatting.Rules.AbstractFormattingRule> GetFormattingRules(Document document);
```

Comment:
>Formatting.Rules [](start = 39, length = 16)

nit: move to using?

Reply:
Name conflict. Resolved in a follow-up that I'll push later.